spec/models/card_spec.rb

```diff
@@ -185,4 +185,22 @@
     check_result = card.check_translation('RoR')
     expect(check_result[:state]).to be false
   end
+
+  describe '.pending_cards_notification' do
+    let(:user1) { create(:user_with_one_block_and_one_card) }
+    let(:user2) { create(:user_with_one_block_and_one_card, email: 'another@mail.com') }
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.